readme.md

```diff
@@ -23,6 +23,7 @@ Create an ```.env``` file with the following sensitive information. Replace the
 ```
 NODE_ENV=development
 DEBUG_MODE=3
+LANG=en
```

Comment:
Consider Changing all instances on LANG to LANGUAGE so it will be easily identified in the codebase and the .ENV file

Reply:
ok changes have been made.